data/gegno/gegno intro missions.txt

```diff
@@ -0,0 +1,565 @@
+		"reputation: Gegno Vi" = -100
+		"reputation: Gegno Scin" = -100
+		event "gegno: unified defense"
+		conversation
+			`As soon as you land your ship, you receive an incoming hail from a Quarg that sounds very displeased. "It was incredibly unwise to engage in the unnecessary destruction of noncombatants belonging to a species not of your own. A species that is still learning from themselves is vulnerable to outside influence, and your aggressive actions against civilians of the Gegno have resulted in needless events.`
+			choice
+				`	"It was an accident!"`
+				`	"What are you talking about?"`
+					goto "what"
+				`	"I do not care."`
+					goto "care"
+			`	"It is greatly unlikely for violence upon unarmed victims to be accidental. Regardless of the cause, the effect has already occurred. The Gegno have banded together under one flag in an effort to deter any more outside interference. You are no longer welcome in their space, and you are advised to not visit again, lest the Gegno show you the same belligerent behavior you have exhibited to their citizens."`
+				accept
+			label "what"
+			`	"Your behavior has caused the Gegno to unite their fleets and deter any more presence from outsiders. It is a great shame. The time they would use to resolve their inner conflicts is now wasted upon hostility towards strangers due to your naivety. You are no longer welcome in Gegno space, and if you dare visit again, the Gegno shall demonstrate the same treatment you gave their citizens."`
+				accept
+			label "care"
+			`	The Quarg makes an unpleasant sound - one you have not heard from any Quarg before. "Should you be seen in Gegno space once more, expect the same sentiment."`
+				accept
+	npc
+		government "Gegno Vi"
+		to spawn
+			has "event: gegno: unified defense"
+		to despawn
+			has "dead augen"
+		personality heroic vindictive staying
+		system "Heutesl"
+		ship "Augen" "Vytii Yrrlausei"
+
+
+				
+# Gegno Hunter Fleets
+
+mission "Gegno Hunter Fleets"
+	landing
+	invisible
+	repeat
+	to offer
+		has "Gegno Genocide Defense: offered"
+		not "Gegno Hunter Fleets"
+	npc
+		government "Gegno"
+		personality nemesis heroic vindictive
+		system
+			government "Gegno"
+		fleet
+			names "gegno united"
+			variant 
+				"Gneiss" 3
+				"Hornfel" 2
+				"Hornfel (Heavy Weapons)" 2
+				"Eclogite" 5
+				"Feldspar" 3
+				"Halite" 18
+				"Felsic" 3
+				"Coesite" 5
+	npc
+		government "Gegno"
+		personality nemesis heroic vindictive
+		system
+			government "Gegno"
+		fleet
+			names "gegno united"
+			variant
+				"Feldspar" 2
+				"Halite" 12
+				"Gneiss" 4
+				"Felsic" 4
+				"Coesite" 4
+				"Eclogite" 3
+	npc
+		government "Gegno"
+		personality nemesis heroic vindictive
+		system
+			government "Gegno"
+		fleet
+			names "gegno united"
+			variant
+				"Feldspar" 3
+				"Halite" 18
+				"Gneiss" 2
+				"Hornfel" 3
+				"Hornfel (Heavy Weapons)"
+				"Eclogite" 3
+				"Coesite" 6
+
+
+
+# Gegno Vi Duels
+
+mission "Gegno Vi Duels"
+	minor
```

Comment:
So it doesn't interfere and bug out other missions being a passive mission. Even if it's invisible, repeat missions like this can possibly interrupt or create issues with others. This was discussed quite a bit and we decided this was best.